articles/install-guide/jupyter.md

```diff
@@ -33,7 +39,7 @@ dotnet iqsharp install
 
 ## Usage ##
 
-Once installed, start jupyter notebook from the command line using:
+Once the `Q#` Jupyter kernel is installed, start jupyter notebook from the command line using:
 
 ```Command Prompt
 jupyter notebook
```

Comment:
GitHub doesn't let me leave this comment in a proper place: should the link on line 51 point to Binder for an interactive version of this notebook?